sdks/java/core/src/main/java/org/apache/beam/sdk/io/LocalFileSystem.java

```diff
@@ -43,10 +54,21 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(LocalFileSystem.class);
 
+  private static final Metadata[] EMPTY_METADATA = new Metadata[0];
+
   LocalFileSystem() {
```

Comment:
LocalFileSystemRegistrar is non-testing code, so I think we don't want VisibleForTesting annotation.